### PR TITLE
plan: make explain result more explicit

### PR DESCRIPTION
**Pull request by hanfei1991**
Print more info for index scan and table scan.
@shenli @coocood @zimulala @XuHuaiyu @tiancaiamao @winoros  PTAL

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: executor/explain_test.go [hunk 1]
```diff
@@ -184,61 +200,75 @@ func (s *testSuite) TestExplain(c *C) {
...
+                ],
+                "filter conditions": null
+            }
         }
     ]
 }`,
 		},
 		{
-			"select count(b.b) from t a, t b where a.a = b.a group by a.b",
+			"select count(b.c2) from t1 a, t2 b where a.c1 = b.c2 group by a.c1",
```
Comment (ngaut) [inline executor/explain_test.go]: We should clean up the previous table.

Comment (hanfei1991) [inline executor/explain_test.go]: I will clean the tables at the end for every test in another pr.

**Comment by ngaut** [comment]
LGTM

**Pull request merged**